scanner/src/lib/dataproducts/battery/CarbonFootprint_v0_1.svelte

```diff
@@ -0,0 +1,74 @@
+<script lang="ts">
+  /*
+    Docs:
+
+    https://github.com/ioxio-dataspace/sandbox-definitions/blob/main/src/DigitalProductPassport/Battery/CarbonFootprint_v0.1.py
+
+    https://definitions.sandbox.ioxio-dataspace.com/definitions/DigitalProductPassport/Battery/CarbonFootprint_v0.1
+
+    https://gateway.sandbox.ioxio-dataspace.com/docs#/Data%20Products/DigitalProductPassport_Battery_CarbonFootprint_v0_1_DigitalProductPassport_Battery_CarbonFootprint_v0_1_post
+     */
+
+  import { formatNumber } from "$lib/common"
+  import DataRow from "$components/DataRow/index.svelte"
+
+  type CarbonFootprint = {
+    preProductionFootprint: number
+    mainProductionFootprint: number
+    referenceMaterial: string
+  }
+
+  export let data: {
+    conformityDeclaration: string
+    carbonFootprint: CarbonFootprint
+  }
+</script>
+
+<article>
+  {#if data.conformityDeclaration}
+    <DataRow label="Conformity Declaration" value={data.conformityDeclaration} />
+  {/if}
+  <div class="divider" />
+  {#if data.carbonFootprint}
+    <div class="title no-bottom-margin">Carbon Footprint</div>
+    <div class="subtitle">
+      The details of the carbon footprint for the battery production phases
+    </div>
+    <DataRow
+      label="Pre Production Footprint"
+      value={formatNumber(data.carbonFootprint.preProductionFootprint, "kg per kWh")}
+    />
+    <DataRow
+      label="Main Production Footprint"
+      value={formatNumber(data.carbonFootprint.mainProductionFootprint, "kg per kWh")}
+    />
+    <DataRow label="Reference Material" value={data.carbonFootprint.referenceMaterial} />
+  {/if}
```

Comment:
Asked for incorrect and inconsistent capitalization to be changed to be correct and consistent with other fields, so `Carbon footprint`, `Pre-production footprint`, and so on